### DISCUSSION TITLE
Docker names and content in deployment (deploy-compose.yml) are wrong and misleading

### DISCUSSION THREAD

**Discussion by ido777** [danny-avila/LibreChat#2087] [Feature Requests & Suggestions]
### What happened?

Containers                                                            

-    IMAGE=librechat-client
-    NAME= LibreChat-NGINX
-    **Expected Content  = frontend + nginx**    
-    **Actual Content  = nginx** 

---

-    IMAGE=ghcr.io/danny-avila/librechat-dev-api:latest
-    NAME= LibreChat-API
-    **Expected Content  = updated backend **    
-    **Actual Content  =  frontend + backend from latest github docker** 

                                        

### Steps to Reproduce

inspect the files in the docker or read the docker file to understand what happens here 

# Nginx setup might have been 

> FROM nginx:1.21.1-alpine AS prod-stage
> COPY ./client/nginx.conf /etc/nginx/conf.d/default.conf
> **COPY --from=client-build /app/client/dist /usr/share/nginx/html**
> CMD ["nginx", "-g", "daemon off;"]

To comply with the current names.
This is not only misleading names, but moreover as of today Dockerfile.multi creates 2 different images since the nginx image does ot take anything from the above layer so if you run it to the end it will not give you the client + API at all, which is ANOTHER BUG

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fuegovic** [comment]
LibreChat-Nginx is indeed just Nginx+a custom config file in the librechat folder. Would using simply "Nginx" be a more suitable name? Maybe...

LibreChat uses a monolithic architecture, which is why the API container encompasses both frontend and backend components.

**Reply by ido777** [reply]
Added my suggestion, WDYT?

**Comment by ido777** [comment]
Here is what I suggest 

-    IMAGE= Nginx
-    NAME= NGINX-Server
-    **Expected Content  = nginx + nginx configuration**    
-    **Actual Content  = nginx + nginx configuration** 

---

-    IMAGE=ghcr.io/danny-avila/librechat-dev-api:latest
-    NAME= Dockerized-App
-    **Expected Content  = frontend + backend pre-packed in this docker -regardless of my source code **    
-    **Actual Content  = frontend + backend pre-packed in this docker -regardless of my source code **

**Reply by fuegovic** [reply]
"Dockerized-App" isn't a good container name, way too generic. I run a dozen "Dockerized-App" in my personal deployment 😉
